metabrainz/templates/index/privacy.html

```diff
@@ -28,16 +160,11 @@ <h2>{{ _('MetaBrainz donors') }}</h2>
     donation, unless you chose to be anonymous during the donation process.') }}
   </p>
 
-  <h2>{{ _('Web and FTP access logs (all users)') }}</h2>
+  <h2>{{ _('Exceptions') }}</h2>
   <p>
-    {{ _('Like just about all web sites, we keep logs of all web requests made
-    against our servers. These logs include the usual stuff: your IP address,
-    your browser\'s "User-Agent" string, and which page you requested.') }}
-  </p>
-  <p>
-    {{ _('Reasonable exceptions may apply to the above policy, for example to comply
-    with applicable laws. The MusicBrainz/MetaBrainz server administrators
-    (about three people in all) can of course see any information on the system
-    they want to, but to be honest they\'re probably not interested enough to look.') }}
+    Please note: Reasonable exceptions may apply to the above policy, for example to comply with applicable laws. The MetaBrainz 
+    server administrators (about five people in all) can of course see any information on the system they want to, but to be honest 
+    we're probably not interested enough to look.
```

Comment:
This section is formatted in [the original/current version on MB.o/doc/](https://wiki.musicbrainz.org/About/Privacy_Policy?oldid=63692#Exceptions). Should that formatting be mirrored here?